warehouse/http.py

```diff
@@ -26,7 +28,12 @@ def __call__(self, request):
             return session
         except AttributeError:
             request.log.debug("creating new session")
+
+            adapter = HTTPAdapter(max_retries=1)
```

Comment:
Will this affect any other remote calls as well? Seems like a not terrible idea overall, just curious what the impact may be.

Reply:
It will affect anything that uses ``request.http``, which is currently just the bit of code that fetches the PEM from SNS to verify SNS messages from SES.